halo2_proofs/Cargo.toml

```diff
@@ -37,8 +37,24 @@ rand = "0.8"
 rand_core = { version = "0.6", default-features = false }
 blake2b_simd = "1"
 pairing = { git = 'https://github.com/appliedzkp/pairing', package = "pairing_bn256", "tag" = "v0.1.1"}
+pairing-bn256 = {package = "pairing_ce", version = "0.24.*" }
 subtle = "2.3"
 cfg-if = "0.1"
+ark-std = { version = "0.3", features = ["print-trace"] }
+
+log = "0.4.8"
+thiserror = "1.0.10"
+crossbeam-channel = "0.5.0"
+crossbeam = {version = "0.7", optional = true}
+lazy_static = {version = "1", optional = true}
+futures = {package = "futures", version = "0.3", default_features = false, features = ["executor"]}
+num_cpus = "1"
+env_logger = "0.8.1"
+
+# gpu feature
+rust-gpu-tools = { version = "0.3.0", optional = true }
+ff-cl-gen = { path = "../../ff-cl-gen", version = "0.2.0", optional = true }
```

Comment:
It seems that this dependency is specified as a relative path but it's not included in the PR.  Does it work with upstream `ff-cl-gen `?
This is currently making the github actions fail.

Reply:
Yes, the intention of this PR is to call response from you to evaluate the feasibility of the solution based on the performance data. 

If you think it is feasible, we will quickly refactor the code to pass the CI.

Reply:
So should we actually review it @bchyl ?? If so, give us a day or two and we'll have it.

Reply:
sorry for the noise.
In this case, I will temporarily close this PR and re-submit a new  after CI passed for refactor code.

thank you very much.

Reply:
refer to  [dependencies tmp](https://github.com/privacy-scaling-explorations/halo2/pull/79/files#r904476006)

> It seems that this dependency is specified as a relative path but it's not included in the PR. Does it work with upstream `ff-cl-gen `? This is currently making the github actions fail.